kolibri/core/bookmarks/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.serializers import ModelSerializer
+
+from .models import Bookmark
+
+
+class BookmarksSerializer(ModelSerializer):
```

Comment:
I know this is not currently how we've done it, but I'd be perfectly happy to keep the serializer alongside the viewset class that it's used in, I don't see much value in how we have structured them right now.